lib/util/viewer.js

```diff
@@ -156,3 +156,53 @@ export function stopTimeComparator (a, b) {
   const bTime = b.serviceDay + b.scheduledDeparture
   return aTime - bTime
 }
+
+/**
+  * Find the stop time corresponding to the first departure
+  * (the closest departure past the current time).
+  */
+export function getFirstDepartureFromNow (mergedStopTimes) {
```

Comment:
This arg should probably just be stopTimes rather than mergedStopTimes.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/ced7d9221f0ad6ada60563f452610feb8f9ed297.